pilot/docker/envoy_pilot.yaml.tmpl

```diff
@@ -45,6 +45,52 @@ static_resources:
     http2_protocol_options: {}
```

Comment:
This file is no longer used in the new installer - we have a config map that can be modified by user or customized.